### PR TITLE
My First Commit!

### PR DESCRIPTION
**Pull request by AbhinavChede**
This is my update of the people's page of the website.

### CODE REVIEW

File: people.md [hunk 1]
```diff
@@ -48,6 +48,17 @@ undergrad:
...
+high school student:
+  - title: Abhinav Chede
+    image_path: assets/images/people/zx_labphoto.jpg
+    excerpt: >
+      I am an high school student at BASIS Chandler in Chandler, AZ. I am a new member of the Qiyun lab. I am interested to learn about microbiology and bioinformatics. I am still exploring possible future careers with an interest in biomedical science.
+      <br /><br />
+      [<i class='fa fa-envelope-square fa-2x'></i>](mailto:abhinavchede@gmail.com)&nbsp;&nbsp;
+      [<i class='fab fa-github-square fa-2x'></i>](https://github.com/AbhinavChede)&nbsp;&nbsp;
+      [<i class='fab fa-linkedin fa-2x'></i>](https://www.linkedin.com/in/abhinav-chede-012777206/)&nbsp;&nbsp;
+      [<i class='ai ai-cv-square ai-2x'></i>](assets/files/Zhu_Xing_resume2021.pdf)
```
Comment (qiyunzhu) [inline people.md:60]: Please update the CV file.

**Review by qiyunzhu — CHANGES_REQUESTED** [review]
Good very good. Welcome on board! I made a minor comment.

File: people.md (hunk 1 above)
Comment (AbhinavChede) [reply people.md]: I have updated my CV!

**Review by qiyunzhu — CHANGES_REQUESTED** [review]
Very good! Now you just need to add a photo, and everything should be good then.

**Review by qiyunzhu — APPROVED** [review]
Looks perfect!

**Pull request merged**